packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -16,7 +16,7 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
 
-Widget buildInputDecorator({
+Widget buildInputDecoratorM2({
```

Comment:
Maybe add a TODO here and point to https://github.com/flutter/flutter/issues/139076? Just to make sure that it's clear that we still intend to migrate these tests to M3. Or maybe down where `runAllM2Tests` is called.